presto-main/src/main/java/com/facebook/presto/sql/analyzer/Analysis.java

```diff
@@ -668,6 +676,16 @@ public void addEmptyColumnReferencesForTable(AccessControl accessControl, Identi
         return tableColumnReferences;
     }
 
+    public Map<AccessControlInfo, Map<QualifiedObjectName, Set<String>>> getUtilizedTableColumnReferences()
```

Comment:
Generally we rarely return mutable containers in Presto codebase. So instead of have this method and call `getUtilizedTableColumnReferences.put(...)`, it's preferable to introduce a function `addUtializedTableColumnRefereces(...)`. Correspondingly, it's probably better for the next function to return `ImmutableMap.copyof(...)`.

Reply:
Done